### DISCUSSION TITLE
Repeat event based on moon phases

### DISCUSSION THREAD

**Discussion by dlquaife** [vigoren/foundryvtt-simple-calendar#571] [Ideas]
I'll start by saying this module is great! Started converting my custom calendar that I started on fantasy-calendar.com but am missing one thing that kind of kills it for me. My world has two moons. Every so often those moons are full at the same time. On the aforementioned site I am able to repeat an even based conditional statements of the moon phases (IF moon 1 full AND moon 2 full, THEN repeat event). Having this capability would enable me to use the simple calendar module and further my goal of having everything I need in Foundry for a One Stop Shop gameplay experience. Thank you for your consideration.

**Comment by Issacar13** [comment]
I'm running a game in the pathfinder world and they have such events has well.
